cirq/ops/common_channels.py

```diff
@@ -184,6 +195,700 @@ def depolarize(p: float) -> DepolarizingChannel:
+        rate gamma at zero temperature
+
+        This channel evolves a density matrix via
+            \rho -> M_0 \rho M_0^\dagger + M_1 \rho M_1^\dagger
+
+        With
+            M_0 = \begin{bmatrix}
+                    1 & 0  \\
+                    0 & \sqrt{1 - \gamma}
+                  \end{bmatrix}
+            M_1 = \begin{bmatrix}
+                    0 & \sqrt{\gamma} \\
+                    0 & 0
+                  \end{bmatrix}
+
+        Args:
+            gamma: The damping rate
+
+        Raises:
+            ValueError: is gamma is not a valid probability.
+        """
+
+        def validate_probability(p, p_str):
+            if p < 0:
+                raise ValueError('{} was less than 0.'.format(p_str))
+            elif p > 1:
+                raise ValueError('{} was greater than 1.'.format(p_str))
+            return p
+
+        self._gamma = validate_probability(gamma, 'gamma')
+        self._delegate = GeneralizedAmplitudeDampingChannel(1.0, self._gamma)
+
+    def _channel_(self) -> Iterable[np.ndarray]:
+        # just return first two kraus ops, we don't care about
+        # the last two.
+        return list(self._delegate._channel_())[:2]
+
+    def _eq_tuple(self):
+        return (AmplitudeDampingChannel, self._gamma)
+
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return NotImplemented
+        return self._eq_tuple() == other._eq_tuple()
+
+    def __hash__(self):
+        return hash(self._eq_tuple())
+
+    def __ne__(self, other):
+        return not self == other
+
+    def __repr__(self) -> str:
+        return 'cirq.amplitude_damp(gamma={!r})'.format(self._gamma)
+
+    def __str__(self) -> str:
+        return 'amplitude_damp(gamma={!r})'.format(self._gamma)
+
+    def _circuit_diagram_info_(
+        self, args: protocols.CircuitDiagramInfoArgs
+    ) -> str:
+        return 'AD({!r})'.format(self._gamma)
+
+
+def amplitude_damp(gamma: float) -> AmplitudeDampingChannel:
+    """
+    Returns an AmplitudeDampingChannel with the given damping rate
+
+    This channel evolves a density matrix via
+            \rho -> M_0 \rho M_0^\dagger + M_1 \rho M_1^\dagger
+
+    With
+        M_0 = \begin{bmatrix}
+                1 & 0  \\
+                0 & \sqrt{1 - \gamma}
+              \end{bmatrix}
+        M_1 = \begin{bmatrix}
+                0 & \sqrt{\gamma} \\
+                0 & 0
+              \end{bmatrix}
+
+    Args:
+        gamma: the damping rate
+
+    Raises:
+        ValueError: if gamma is not a valid probability.
+    """
+    return AmplitudeDampingChannel(gamma)
+
+
+class PhaseDampingChannel(raw_types.Gate):
+    """Dampen qubit phase.
+
+    This channel models phase damping which is the loss of quantum
+    information without the loss of energy.
+    """
+
+    def __init__(self, gamma) -> None:
+        """The phase damping channel.
+
+        Construct a channel that enacts phase damping at the rate
+        given by gamma
```

Comment:
I wouldn't call this a rate.  Perhaps just say 

 Construct a channel that enacts phase damping constant gamma.

(If you don't care about the measurement info this is really a probabilistic unitary with a probability of flip given by (1+sqrt(1-gamma)) / 2

Reply:
fixed, changed to be exactly "Construct a channel that enacts phase damping constant gamma."